Gopkg.toml

```diff
@@ -79,3 +79,7 @@
 [prune]
   go-tests = true
   unused-packages = true
+
+[[constraint]]
+  name = "github.com/bmatcuk/doublestar"
+  version = "1.1.1"
```

Comment:
Is the constraint necessary?  Can't we use the latest and greatest?  In general, every entry in this file needs a justification.